src/api/urls.py

```diff
@@ -10,5 +10,7 @@
 
 app_name = "api"
 urlpatterns = [
-    path("", include(router.urls))
+    path("", include(router.urls)),
+    path('api/health/', views.health_check, name='health_check'),
+    path('api/ready/', views.readiness_check, name='readiness_check'),
```

Comment:
Incorrect path. If you use `app/health` path inside app_name api, final path will be `api/api/health`

```suggestion
    path("", include(router.urls)),
    path('health', views.health_check, name='health_check'),
    path('ready', views.readiness_check, name='readiness_check'),
```